apps/cnquery/cmd/builder/builder.go

```diff
@@ -570,14 +570,14 @@ func azureProviderCmd(commonCmdFlags commonFlagsFn, preRun commonPreRunFn, runFn
 		},
 	}
 	commonCmdFlags(cmd)
-	cmd.Flags().String("tenant-id", "", "directory (tenant) ID of the service principal")
-	cmd.Flags().String("client-id", "", "application (client) ID of the service principal")
-	cmd.Flags().String("client-secret", "", "secret for application")
-	cmd.Flags().String("certificate-path", "", "path to certificate that's used for certificate-based authentication in PKCS 12 format (pfx)")
-	cmd.Flags().String("certificate-secret", "", "passphrase for certificate file")
-	cmd.Flags().String("subscription", "", "the Azure subscription ID to scan")
-	cmd.Flags().String("subscriptions", "", "a comma-separated list of Azure subscriptions to include")
-	cmd.Flags().String("subscriptions-exclude", "", "a comma-separated list of Azure subscriptions to exclude")
+	cmd.Flags().String("tenant-id", "", "Directory (tenant) ID of the service principal")
+	cmd.Flags().String("client-id", "", "Application (client) ID of the service principal")
+	cmd.Flags().String("client-secret", "", "Secret for application")
+	cmd.Flags().String("certificate-path", "", "Path to certificate that's used for certificate-based authentication in PKCS 12 format (pfx)")
+	cmd.Flags().String("certificate-secret", "", "Passphrase for certificate file")
```

Comment:
If this is for the certificate, as the name suggests, then how about:

```suggestion
	cmd.Flags().String("certificate-secret", "", "Passphrase for the authentication certificate file")
```